extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -277,8 +277,15 @@ export class JupyterServerInstallation {
 		};
 	}
 
-	private isPythonRunning(pythonInstallPath: string): Promise<boolean> {
-		return Promise.resolve(false);
+	private async isPythonRunning(installPath: string, existingPython: boolean): Promise<boolean> {
+		if (process.platform === constants.winPlatform) {
+			let pythonExe = JupyterServerInstallation.getPythonExePath(installPath, existingPython);
+			let cmd = `powershell.exe -NoProfile -Command "& {Get-Process python | Where-Object {$_.Path -eq '${pythonExe}'}}"`;
+			let processInfo = await this.executeBufferedCommand(cmd);
```

Comment:
can we put this in a try/catch?

Reply:
do people remove powershell from their path, by the way? Trying to think of ways in which this could fail